ksp_plugin_adapter/localization/zh-cn.cfg

```
@@ -73,16 +77,22 @@ Localization {
     #Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating = <<1>>的质心已固定，\n水平面为它的赤道。\n这个参考系不旋转。
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection = <<1>>的质心已固定，\n水平面为绕着<<3>>的轨道。\n这个参考系跟随轨道旋转，面向着<<3>>的方位不变。
     #Principia_ReferenceFrameSelector_Description_BodySurface = <<1>>已固定，\n水平面为它的赤道。\n这个参考系与<<1>>共同旋转。
+    // TODO(Al₂Me₆): Translate these.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = The centres of <<A:3>> and <<A:1>> are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = The centre of <<A:3>> and the barycentre of the <<nom:1>>–<<nom:5>> system are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = The centre of <<A:3>> and the barycentre of the <<nom:1>> system are fixed.\nThe plane of their orbit is horizontal.\nThe reference frame rotates with the orbit and pulsates to keep the <<nom:3>>–<<nom:1>> distance constant.
```

```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = <<3>>与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = <<3>>的质心与<<1>>-<<5>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = <<3>>的质心与<<1>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
```

(Yes, both 已 and 以 are intentional this time XD)